### PR TITLE
Job history retention documentation

### PR DESCRIPTION
**Pull request by kandadaboggu**

### CODE REVIEW

File: job-history-replication.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+---
+
+# Job history replication
+Workato can store the job history of all your recipes to an S3 bucket. 
+When enabled, we will create a JSON file for each job. This file will 
+contain job summary, job report, and details of all executed recipe steps. 
+The file size varies from 1kB to 1MB based on the details present in the line 
+execution logs.
+
+This is an add-on feature. Talk to your account representative to enable this
```
Comment (graceliching) [inline job-history-replication.md]: This is an add-on feature. Please reach out to us at +1 (844) 469-6752 or talk to your account representative to enable job history replication in your account.

File: job-history-replication.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+When enabled, we will create a JSON file for each job. This file will 
+contain job summary, job report, and details of all executed recipe steps. 
+The file size varies from 1kB to 1MB based on the details present in the line 
+execution logs.
+
+This is an add-on feature. Talk to your account representative to enable this 
+feature in your account.
+
+## Replication setup
+Click on the `Job history replication` menu in the `Account Settings` screen.
```
Comment (graceliching) [inline job-history-replication.md]: Click on the `Job history replication` tab in your `Account Settings`.

File: job-history-replication.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+![Job history replication setup](/assets/images/job-history-replication/settings.png)
+*Job history replication setup.*
+
+
+| Name             | Description           |
+|------------------|-----------------------|
+| Enable S3 replication   | Choose 'yes' to replicate job history data in Amazon S3. |
+| S3 connection | Choose an Amazon S3 connection that has region and bucket. The connection should have read/write access to the bucket.                    |
+| Store line by line execution logs     | Store input, output and error details of all executed lines.                    |
```
Comment (graceliching) [inline job-history-replication.md]: Choose 'yes' to store input, output and error details of all executed lines.

File: job-history-replication.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+Workato creates a JSON file for each completed or failed job in your account. 
+
+### File name
+
+The file path and name format is as follows:
+
+
+| File Part             | Description           |
+|------------------|-----------------------|
+| path        | _user_id/jobs/recipe_id/YYYYMMDD/**formatted_job_id**/_<br><br>_**formatted_job_id**_ is the job id expanded in to a 21 digit number(left padded with 0s) and split in to 3 character fragments separated by `/`. <br><br>_Eg: Job id `1` is formatted as `000/000/000/000/000/000/001`_ |
```
Comment (graceliching) [inline job-history-replication.md]: <br>_Eg: Job id `100` is formatted as `000/000/000/000/000/000/100`_ |

- changed to 100 to match the examples below

File: job-history-replication.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+Eg: _`5234/jobs/234/20180521/000/000/000/000/000/000/100/5234-234-100-20180521000000-succeeded.json`_
+
+### File content
+
+The job log file contains the job summary data, job custom report data and job execution logs.
+
+```javascript
+{
+	"id": 100,
+	"flow_id": 100,
```
Comment (graceliching) [inline job-history-replication.md]: "flow_id": 234,

File: job-history-replication.md (hunk 3 above)
Comment (kandadaboggu) [reply job-history-replication.md]: New text for this field has changed.

**Pull request merged**